@xen-orchestra/lite/src/components/NoData.vue

```diff
@@ -0,0 +1,32 @@
+<template>
```

Comment:
The component name: `NoData` can be confusing since the absence of data is not specifically due to an error.
`ErrorNoData`?

cc/ @ByScripts @pdonias

Reply:
As it currently stands in Figma, an error occurs if we don't collect data and I find this to be logical.